File: First_Year/II/PLA/labs/tema2/problema2/prb2.py
AX = 5
BX = 13
DI = 2
a = 13
y = 4

def power(base, exponent):
    result = 1
    for _ in range(exponent):
        result *= base
    return result

sum_result = 0
for j in range(1, 20):
    sum_result += ((j * AX) / (power(a, y)) - (power(BX, DI))) / (20 + j)

print(f"The result is: {sum_result}")

File: First_Year/II/PLA/labs/tema2/problema2/prb2.c
#include <stdio.h>
#include <math.h>

int main() {
    double AX = 5;
    double BX = 13;
    double DI = 2;
    double a = 13;
    double y = 4;

    double sum_result = 0;
    for(int j = 1; j < 20; j++)
        sum_result += ((j * AX) / pow(a, y) - pow(BX, DI)) / (20 + j);

    printf("The result is: %f\n", sum_result);

    return 0;
}
